### ISSUE TITLE
[Feature Request]: Add preventDefault to the debugger buttons of the interactions addon.

### ISSUE THREAD

**Issue by santiagopuentep** [storybookjs/storybook#21259]
### Is your feature request related to a problem? Please describe

The debugger buttons of the interactions addon make elements in the story lose focus, making it impossible to do a step by step debugging in stories where an element has to be in focus for some other element to be clicked using `testing-library`.

### Describe the solution you'd like

The debugger buttons should have `event.preventDefault()` called on the _"pointerdown"_ event on them to prevent other elements from losing focus.

### Describe alternatives you've considered

I temporarily solved this by creating an event listener attached to the `document` that checks if one of the interactions addon debugger buttons was pressed and call `preventDefault()` manually, but it's a hack with a while loop.

### Are you able to assist to bring the feature to reality?

no
